src/style.css

```
@@ -0,0 +1,66 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  width: 100%;
+  height: 100%;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.stars__block {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.stars {
+  position: relative;
+  display: flex;
+  width: 100%;
+}
+
+.stars__star {
+  position: relative;
+  display: block;
+  width: 16px;
+  height: 16px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars.stars--5 .stars__star {
+  background-image: url("images/star-active.svg");
+}
+
+.stars.stars--1 .stars__star:nth-child(1) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars.stars--2 .stars__star:nth-child(1),
+.stars.stars--2 .stars__star:nth-child(2) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars.stars--3 .stars__star:nth-child(1),
+.stars.stars--3 .stars__star:nth-child(2),
+.stars.stars--3 .stars__star:nth-child(3) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars.stars--4 .stars__star:nth-child(1),
+.stars.stars--4 .stars__star:nth-child(2),
+.stars.stars--4 .stars__star:nth-child(3),
+.stars.stars--4 .stars__star:nth-child(4) {
+  background-image: url("images/star-active.svg");
+}
```

```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
  background-image: url("images/star-active.svg");
}
```
styles for active star can be more compact and simple

---

Fix styles in new commit